core/cat/mad_hatter/plugin.py

```diff
@@ -241,6 +241,16 @@ def _load_manifest(self):
         meta["thumb"] = json_file_data.get("thumb", "")
         meta["version"] = json_file_data.get("version", "0.0.1")
 
+        # Core compatibility
+        compatibility = json_file_data.get(
+            "compatibility",
+            {
+                "min_version": "not specified",
+                "max_version": "not specified",
```

Comment:
maybe empty string as default value, we made a mess in the past with a different default for every key XD